pages/signin.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
보통 스크린 리더가 이 lang attritube를 보고 음성을 변환하기 때문에
요부분은 웹 접근성 차원에서 korean 으로 변경해주시는게 좋아요! 

```suggestion
<html lang="ko">
```